docs/testing/releases/380.md

```diff
@@ -0,0 +1,21 @@
+## Testing notes and ZIP for release 3.8.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/5516903/woocommerce-gutenberg-products-block.zip)
+
+## The following changes affect the feature plugin only.
+
+### Show the phone number field in the billing section when shipping is disabled in settings. ([3376](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3376))
+
+Moves the rendering of address fields to a single file so that PhoneNumber (new component) can be rendered in either billing or shipping, depending on visibility, without duplication.
+
+0. Ensure the phone number field is enabled in block settings, then test 2 cases:
+1. With shipping enabled, ensure phone number displays beneath the shipping step.
+2. With shipping disabled (you can delete all rates and zones to edit general settings), ensure phone number displays beneath the billing step.
+3. In both scenarios the values should change and be saved as billing information.
+
+## The following changes affect feature plugin and WooCommerce package.
+
+### Change delimiter used for spltChunks from tilde to hyphens. ([3360](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3360))
+
+0. Complete a build and smoke test block inserter - atomic blocks would be affected if this broke anything.
```

Comment:
> atomic blocks

It might be good to specify that in Core those blocks are only accessible when editing the All Products block.